netci.groovy

```diff
@@ -968,18 +968,18 @@ def static getDockerImageName(def architecture, def os, def isBuild) {
         }
         else if (architecture == 'armem') {
             if (os == 'Ubuntu') {
-                return "microsoft/dotnet-buildtools-prereqs:ubuntu-14.04-cross-0cd4667-20172211042239"
```

Comment:
Not sure you want to touch the 'armem' case.

Actually, I think we might want to delete 'armem' entirely now that we are running on hardware.

Reply:
Does anybody mind to delete `armem`, @dotnet/arm32-contrib?
if no one is against then I will delete it entirely in a separate PR.